### ISSUE TITLE
Authorization Code call fails because of extra characters appended to state parameter.

### ISSUE THREAD

**Issue by adamcjohnson** [jeyben/IOSLinkedInAPI#72]
The authorization code response contains extra characters appended to the `state` parameter, which therefore makes the request fail. 

I've resolved this issue in: https://github.com/jeyben/IOSLinkedInAPI/pull/71

**Comment by NickChayka** [comment]
I think, this is not a good idea - use containsString instead isEqualToString. 

My solution:

in LIALinkedInAuthorizationViewController.m

line 134:

```
// urlString = [[urlString componentsSeparatedByString:@"?"] objectAtIndex:1];
urlString = [[NSURL URLWithString:urlString] query];
```

**Comment by eyalzohar** [comment]
Can you please update your Pod with the latest Master ?

**Comment by santilorenzo** [comment]
Yes, it fixed the error to me too. Can you please make a fork and a PR?
Thank you very much!

**Comment by junestchick** [comment]
Did it fixed ?

**Comment by zfrankz** [comment]
I've made a pull request. All came from the end point used to authenticate.
